### PR TITLE
Path param descriptions

### PR DESCRIPTION
**Pull request by lfportal**
Currently SPOT generates OpenAPI documentation with placeholder text for path param descriptions. This will change this to custom descriptions.

### CODE REVIEW

File: lib/src/parser.ts [hunk 1]
```diff
@@ -238,7 +238,8 @@ function extractEndpoint(
               ? undefined
               : nextNonNamePositionRelative
           ),
-          type: VOID
+          type: VOID,
+          description: 'TODO'
```
Comment (fwouts) [inline lib/src/parser.ts]: Instead of `TODO`, could we make this an empty string?

**Review by fwouts — CHANGES_REQUESTED** [review]
Sweet!

File: lib/src/parser.ts (hunk 1 above)
Comment (lfportal) [reply lib/src/parser.ts]: Yeap

**Review by fwouts — APPROVED** [review]

File: examples/src/multi-file/endpoints/get-user.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ export class Api {
     path: "/users/:userId"
   })
   getUser(
-    @pathParam userId: string
+    @pathParam() userId: string
```
Comment (danieltheodosius) [inline examples/src/multi-file/endpoints/get-user.ts]: should we make the example `multi-file` and `single-file` to be consistent? this one does not have description, while the one in the single file has description

**Review by danieltheodosius — APPROVED** [review]

**Pull request merged**